source/common/config/well_known_names.cc

```diff
@@ -132,6 +132,12 @@ TagNameValues::TagNameValues() {
   // internal state of the regex which performs better.
   addRe2(HTTP_CONN_MANAGER_PREFIX, R"(^listener\..*?\.http\.((<TAG_VALUE>)\.))", ".http.");
 
+  // Extract ext_authz stat_prefix field
+  // cluster.[<cluster>.]ext_authz.[<ext_authz_prefix>.]*
+  addRe2(EXT_AUTHZ_PREFIX, R"(^cluster\.(?:[^\.]+\.)?ext_authz\.((<TAG_VALUE>)\.))");
```

Comment:
Should these be `addTokenized` instead? @jmarantz ? Similar to `addTokenized(GRPC_BRIDGE_METHOD, "cluster.*.grpc.*.$.**");` earlier in the list.

Reply:
Yes that sounds good to me; that will be the fastest option.